packages/webdriverio/src/scripts/isElementClickable.js

```diff
@@ -65,23 +69,30 @@ export default function isElementClickable (elem) {
 
     // is one of overlapping elements the `elem` or one of its child
     function isOverlappingElementMatch (elementsFromPoint, elem) {
-        if (elementsFromPoint.some(elementFromPoint => elementFromPoint === elem || nodeContains(elem, elementFromPoint))) {
+        if (elementsFromPoint.some(function (elementFromPoint) {
+            return elementFromPoint === elem || nodeContains(elem, elementFromPoint)
+        })) {
             return true
         }
 
         // shadow root
         // filter unique elements with shadowRoot
-        let elemsWithShadowRoot = [...new Set(elementsFromPoint)]
-        elemsWithShadowRoot = elemsWithShadowRoot.filter(x => x && x.shadowRoot && x.shadowRoot.elementFromPoint)
+        let elemsWithShadowRoot = [].concat(elementsFromPoint)
+        elemsWithShadowRoot = elemsWithShadowRoot.filter(function (x) {
+            return x && x.shadowRoot && x.shadowRoot.elementFromPoint
+        })
 
         // getOverlappingElements of every element with shadowRoot
         let shadowElementsFromPoint = []
-        for (let shadowElement of elemsWithShadowRoot) {
-            shadowElementsFromPoint.push(...getOverlappingElements(elem, shadowElement.shadowRoot))
+        for (let i = 0; i < elemsWithShadowRoot.length; ++i) {
+            let shadowElement = elemsWithShadowRoot[i]
```

Comment:
if the idea is to make it all ES5 and thus IE11 compatible then the keywords `let` and `const` cannot be used either.

Reply:
@erwinheitzman `let` and `const` are supported by IE11: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let